index.bs

```diff
@@ -5558,8 +5609,9 @@ partial interface MLGraphBuilder {
 
 <details open algorithm>
   <summary>
-    The <dfn method for=MLGraphBuilde id=softplus-noargs>softplus()</dfn> method steps are:
+    The <dfn method for=MLGraphBuilder id=softplus-noargs>softplus()</dfn> method steps are:
```

Comment:
Thanks for the fix. It seems like we should be able to "lint" this - e.g. verify that no methods (etc) are defined "for" a scope that isn't defined in IDL, or something

Reply:
If you add this near the end of tools/lint.mjs it will catch the error:

```js
// Ensure every method dfn is correctly associated with an interface
const interfaces = new Set(
  root.querySelectorAll('dfn[data-dfn-type=interface]').map(e => e.innerText));
for (const dfn of root.querySelectorAll('dfn[data-dfn-type=method]')) {
  const dfnFor = dfn.getAttribute('data-dfn-for');
  if (!dfnFor || !interfaces.has(dfnFor)) {
    error(`Method definition '${dfn.innerText}' for undefined '${dfnFor}'`);
  }
}
```

Reply:
Created #720